src/services/user.ts

```diff
@@ -43,13 +45,19 @@ export async function getAuthUser(): Promise<DocumentData | null> {
  * @param {string} roleType role of the user.
  * @returns Returns nothing.
  */
-export async function createUser({ email }: { email: string }, roleType: string): Promise<void> {
+export async function createUser(
+  { email, first_name, last_name, gender, phone_number, emergency_contact, address }: UserDetail,
+  roleType: string
+): Promise<void> {
   try {
-    await setDoc(doc(db, "users", email), {
-      first_name: "test",
-      last_name: "account",
+    await setDoc(doc(db, "users", email?.trim()?.toLowerCase()), {
```

Comment:
I don't think you'll need a question mark after `trim()`

Reply:
Got it. Removed.